go/vm/lfvm/opcode.go

```diff
@@ -144,6 +144,8 @@ const (
 	// Storage
 	SLOAD
 	SSTORE
+	TLOAD
+	TSTORE
```

Comment:
I think this file was as well the op_to_string map, does it not?

Reply:
Missed this one, thank you.